modules/Core/src/main/java/org/terasology/core/world/CoreBiomes.java

```diff
@@ -35,6 +37,19 @@
      */
     @Override
     public void preBegin() {
+        CoreBiome.DESERT.setLowerLimit(SurfaceTemperatureFacet.class, 0.5f);
+        CoreBiome.DESERT.setUpperLimit(SurfaceHumidityFacet.class, 0.3f);
+        CoreBiome.SNOW.setUpperLimit(SurfaceTemperatureFacet.class, 0.3f);
+        CoreBiome.SNOW.setLowerLimit(SurfaceHumidityFacet.class, 0.5f);
+        CoreBiome.PLAINS.setLimits(SurfaceHumidityFacet.class, 0.3f, 0.6f);
+        CoreBiome.PLAINS.setLowerLimit(SurfaceTemperatureFacet.class, 0.5f);
+        CoreBiome.MOUNTAINS.setLimits(SurfaceHumidityFacet.class, 0.2f, 0.6f);
+        CoreBiome.MOUNTAINS.setUpperLimit(SurfaceTemperatureFacet.class, 0.5f);
+        CoreBiome.FOREST.setLowerLimit(SurfaceHumidityFacet.class, 0.3f);
+        CoreBiome.FOREST.setLimits(SurfaceTemperatureFacet.class, 0.4f, 0.7f);
+        //Beach and Ocean are special hardcoded conditions for now
+        CoreBiome.OCEAN.setLimits(SurfaceTemperatureFacet.class, 1F, 0F);
+        CoreBiome.BEACH.setLimits(SurfaceTemperatureFacet.class, 1F, 0F);
```

Comment:
@RatMoleRat this might be of interest to you - probably a quick chat with @Qwertygiy on this matter can help with the whole temperature thing?